lib/dockerregistry/inventory.go

```diff
@@ -1631,10 +1631,16 @@ func (sc *SyncContext) ExecRequests(
 	// We have to use a WaitGroup, because even though we know beforehand the
 	// number of workers, we don't know the number of jobs.
 	wg := new(sync.WaitGroup)
+
+	var err error = nil
```

Comment:
FYI in Golang just `var err error` does the same thing. If you just declare vars without providing the value, it just gets assigned the default value.